typescript/src/events/guilds/members/guildMemberAdd.ts

```diff
@@ -22,14 +21,19 @@ export class UserEvent extends Event {
 		if (stickyRoles.length === 0) return false;
 
 		// Handle the case the user is muted
-		const [roleID, t] = await member.guild.readSettings((settings) => [settings[GuildSettings.Roles.Muted], settings.getLanguage()]);
-		if (roleID && stickyRoles.includes(roleID)) {
+		const key = GuildSettings.Channels.Logs.MemberAdd;
+		const [channelId, roleId, t] = await member.guild.readSettings((settings) => [
```

Comment:
```suggestion
		const [logChannelId, roleId, t] = await member.guild.readSettings((settings) => [
```

And rename references accordingly. I had to check what the heck `channelId` was, a simple other name makes it easier already.